bikeshed/cli.py

```diff
@@ -579,7 +580,7 @@ def handleDebug(options, extras):
         doc = Spec(inputFilename=options.infile)
         doc.mdCommandLine = metadata.fromCommandLine(extras)
         doc.preprocess()
-        refs = doc.refs.refs[options.linkText] + doc.refs.refs[options.linkText + "\n"]
+        refs = doc.refs[options.linkText] + doc.refs[options.linkText + "\n"]
```

Comment:
I don't like this and I'm not sure if either way is correct. Could only guess that maybe the `getRef` might be a better thing, except didn't dig through that

Reply:
Ooh yeah, that's some old code from back when the ReferenceManager class had a single `refs` member (now it's split across three members). This is obsoleted by the `bikeshed refs` command now, so we should just drop this whole `elif` and remove it from the command's options.